### DISCUSSION TITLE
More Categories

### DISCUSSION THREAD

**Discussion by Murplyy** [elrumo/macOS_Big_Sur_icons_replacements#1386] [Ideas]
I think it would be nice to be able to have more categories to more efficiently sort what you're looking for, and/or categories that group submissions that tend to have a lot of icons like Adobe, Google, Apple Apps, etc.

Some category suggestions:

- Folders / Files
- Apps / Software
- Launchers (ex: Rockstar Launcher, Technic Launcher, CurseForge, Origin, Ubisoft, Adobe CC)
- Tools / Utilities / Extensions (ex: Color Pickers, Pictogram, Keka, Huion, Wacom, Cheat Sheet, Honey)

**Comment by KenGuru-blog** [comment]
Yes, good idea 👍

**Comment by elrumo** [comment]
Good idea, I'll try to add it when I have some more time.

**Reply by elrumo** [reply]
I'll keep this thread posted.
